warrior/Framework/Utils/data_Utils.py

```diff
@@ -1266,13 +1266,15 @@ def resolve_argument_value_to_get_tag_value(datafile, system_name,
     if element_value_in_argument.startswith("tag"):
         tag_name = evaluate_tc_argument_value(element_value_in_argument)
         if tag_name is not False:
```

Comment:
The else condition corresponding to this if should have a warning message that .. could nto find tag with the provided name = <name provide> hence returning the value as such.. or something like this